common/static/common/templates/discussion/new-post.underscore

```diff
@@ -20,13 +20,16 @@
     <% } %>
     <div class="post-field">
         <label class="field-label">
-            <span class="sr"><%- gettext("Title:") %></span>
-            <input aria-describedby="field_help_title" type="text" class="field-input js-post-title" name="title" placeholder="<%- gettext('Title') %>">
+            <span class="field-label-text"><%- gettext("Title:") %></span><input aria-describedby="field_help_title" type="text" class="js-post-title field-input" name="title">
```

Comment:
If this is a required field (assuming it is?) it may be helpful to add `required`. Although, if the form isn't "submitted", it's moot.